docs/ubuntu_install.rst

```diff
@@ -34,18 +35,12 @@ Verify the ``SHA256SUMS`` file and move on to the next step if you see
 
     gpg --verify SHA256SUMS.gpg SHA256SUMS
 
-The next and final step is to verify the Ubuntu image. If you are using
-Linux, use the following command.
+The next and final step is to verify the Ubuntu image.
 
 ::
```

Comment:
Fixed in https://github.com/fowlslegs/securedrop/commit/caca462e2b3fb764587d7d42c3e6d0971fb80981